### PR TITLE
VZ-7637 - remove last remaining manual step (admin cluster URL) in MC pre-registration

### PR DESCRIPTION
**Pull request by desagar**

### CODE REVIEW

File: content/en/docs/troubleshooting/troubleshooting-multicluster.md [hunk 1]
```diff
@@ -61,11 +62,26 @@ $ kubectl logs \
     -n verrazzano-system \
     -l app=verrazzano-application-operator
 ```
-If these logs reveal that there is a connectivity issue, check the admin cluster Kubernetes server address that
-you provided during registration and ensure that it is correct, and that it is reachable from the managed cluster. If it
-is incorrect, then you will need to repeat the managed cluster registration process described in the setup instructions
+If these logs reveal that there is a connectivity issue, then in the case of an installation that includes Rancher on
+the admin cluster, there may have been a problem with Verrazzano pushing registration details or updates to the managed
+cluster. Try exporting and applying the registration manifest to the managed cluster as shown below:
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/troubleshooting-multicluster.md:67]: as shown below: -> as shown:

File: content/en/docs/troubleshooting/troubleshooting-multicluster.md [hunk 2]
```diff
@@ -61,11 +62,26 @@ $ kubectl logs \
...
+# on the admin cluster
+       kubectl get secret \
+       -n verrazzano-mc verrazzano-cluster-managed1-manifest \
+       -o jsonpath={.data.yaml} | base64 --decode > register.yaml
+
+# on the managed cluster
+       kubectl apply -f register.yaml
+```
+
+*Note:* If your installation disabled Rancher on the admin cluster, then check the admin cluster Kubernetes server
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/troubleshooting-multicluster.md:79]: *Note:* -> **NOTE**: (Note bold, colon not bold)

**Review by fredtibbitts — APPROVED** [review]

File: content/en/docs/troubleshooting/troubleshooting-multicluster.md [hunk 3]
```diff
@@ -61,11 +62,26 @@ $ kubectl logs \
...
+address that you provided during registration and ensure that it is correct, and that it is reachable from the managed
+cluster. If it is incorrect, then you will need to repeat the managed cluster registration process described in the setup instructions
```
[the remark this answers is not in the document]
Comment (desagar) [reply content/en/docs/troubleshooting/troubleshooting-multicluster.md:81]: That does not seem to work in the relref - causes an error starting up hugo. I think this is fine either way so I left it as is.

[the remark this answers is not in the document]
Comment (desagar) [reply content/en/docs/troubleshooting/troubleshooting-multicluster.md:81]: never mind. got it to work and pushed.

**Pull request merged**